addons/website/views/website_templates.xml

```diff
@@ -700,7 +700,7 @@
               </t>
               <t t-if="version">
                 <h1><t t-esc="res_company.name"/>
-                    <small>Odoo Version <t t-raw="version.get('server_version')"/></small>
+                    <small>Odoo Version <t t-esc="version.get('server_version')"/></small>
```

Comment:
Why this `t-raw` -> `t-esc` change in the PR?

Reply:
All char fields and no need for `t-raw`

Reply:
for security reasons FYI